src/Microsoft.Framework.Runtime.Abstractions/project.json

```diff
@@ -2,6 +2,12 @@
     "version": "1.0.0-*",
     "description": "ASP.NET 5 interfaces to take advantage of capabilities exposed by the runtime.",
     "compilationOptions": { "warningsAsErrors": true },
+    "dependencies": {
+        "Microsoft.Framework.HashCodeCombiner.Sources": {
```

Comment:
I think you missed the point of this thing. Andrew pretty much summed it up https://github.com/aspnet/dnx/pull/2359#discussion_r35613141